src/osip-core/src/main/java/edu/kit/pse/osip/core/io/files/ServerSettingsWrapper.java

```diff
@@ -1,39 +1,72 @@
 package edu.kit.pse.osip.core.io.files;
 
 import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Properties;
 
 import edu.kit.pse.osip.core.model.base.TankSelector;
 
 /**
  * Wrapper for server settings file where ports of tanks are stored
+ * Key consists of parameterType_TankSelector
+ * @author Maximilian Schwarzmann
+ * @version 1.0
  */
 public class ServerSettingsWrapper {
+    
+    private File propFile;
+    private Properties properties;
+    
     /**
      * Constructor of ServerSettingsWrapper
-     * @param settingsFile The file to save the settings to
+     * @param propFile The file to save the settings to
+     * @throws IOException  Exception in FileInputStream
+     * @throws FileNotFoundException Exception in file param
      */
-    public ServerSettingsWrapper (File settingsFile) {
-        throw new RuntimeException("Not implemented!");
+    public ServerSettingsWrapper(File propFile) throws IOException, FileNotFoundException  {
+        if (propFile == null) {
+            throw new NullPointerException();
```

Comment:
Wäre gut wenn du hier eine Beschreibung in die Exception speicherst, also z.B. "Properties File is null" oder so.